src/platform/silabs/wifi/BUILD.gn

```diff
@@ -151,4 +151,14 @@ source_set("wifi-platform") {
       "${silabs_platform_dir}/wifi/lwip-support/lwip_netif.cpp",
     ]
   }
+
+  # TODO: Once the WifiInterface refactor is done and becomes a class, we will be inject an interface instead of having them in same source_set
```

Comment:
```suggestion
  # TODO: Once the WifiInterface refactor is done and becomes a class, we will inject an interface instead of having them in same source_set
```